knimin/config.txt.example

```diff
@@ -3,9 +3,9 @@ debug = True
 help_email =
 
 [postgres]
-USER =
+USER = postgres
 PASSWORD =
-DATABASE =
+DATABASE = ag_test
```

Comment:
since labadmin is not ag, would it be possible to rename this? i think it would be a great policy going forward to avoid nomenclature that ties this to ag